Cargo.toml

```diff
@@ -39,7 +39,7 @@ clap = "2.33.3"
 combinations = "0.1.0"
 console_error_panic_hook = "0.1.7"
 console_log = { version = "0.2", optional = true }
-flux = { git = "https://github.com/influxdata/flux", tag = "v0.143.1" }
+flux = { git = "https://github.com/influxdata/flux", branch = "master" }
```

Comment:
Is this how you normally update mid Flux release? Or do you just cut a Flux release and always point at a tag?

Reply:
You can do `rev = "abc123"` instead of `branch` - I'm not sure how comfortable I am with a rev over a tag though. I think I'd rather see a flux tag that we reference.